### PR TITLE
Update heading & icon sizes

### PR DESCRIPTION
**Pull request by starryeyez024**
They now use math + global font size variable

### CODE REVIEW

File: themes/cp-theme/red-hat-sass/variables/_typography.scss [hunk 1]
```diff
@@ -29,21 +29,21 @@ $rh-global--FontFamily--monospace:     "Overpass Mono", Consolas, Monaco, 'Andal
...
-// ========================================================================
-// Icons Sizes
-// ========================================================================
-
-$rh-global--icon-size--xl:              rh-size-prem(64px) !default;
-$rh-global--icon-size--lg:              rh-size-prem(48px) !default;
-$rh-global--icon-size--md:              rh-size-prem(32px) !default;
-$rh-global--icon-size--sm:              rh-size-prem(16px) !default;
-$rh-global--icon-size--xs:              rh-size-prem(14px) !default;
+$rh-global--FontSize--heading--xxl:     rh-size-prem($rh-global--font-size-root * 2)    !default;  // 32px
```
Comment (castastrophe) [inline themes/cp-theme/red-hat-sass/variables/_typography.scss]: Should this use  `$rh-global--FontSize` instead?   $rh-global--FontSize maps to `$rh-global--font-size-root` on line 9.

Comment (castastrophe) [reply themes/cp-theme/red-hat-sass/variables/_typography.scss]: Looks good otherwise

Comment (starryeyez024) [reply themes/cp-theme/red-hat-sass/variables/_typography.scss]: updated - @markcaron are you good with this, can we merge?

Comment (markcaron) [reply themes/cp-theme/red-hat-sass/variables/_typography.scss]: I'm good with this change. We should definitely be basing the sizes for headings and icons off FontSize.

**Comment by markcaron** [comment]
Lifelong Gym Touts Monthlies

**Pull request merged**